src/Bicep.LangServer/Snippets/Templates/res-firewall-standard-vwan.bicep

```diff
@@ -0,0 +1,22 @@
+// Azure Firewall Standard SKU for Virtual WAN Hub
+resource /*${1:firewall}*/firewall 'Microsoft.Network/azureFirewalls@2021-05-01' = {
+  name: /*${2:'name'}*/'name'
+  location: /*${3:location}*/'location'
+  properties: {
+    sku: {
+      name: 'AZFW_Hub'

```

Comment:
Can you turn this into a choice tabstop?  Looks like AZFW_Hub, AZFW_VNet are the only options, right?

Reply:
Yes there are two different options on the SKU name. The reason why I haven't made it into a tabstop is because the allowed properties differ between the two choices. My plan was to add two separate snippets, one for VNet and one for VWAN to make it easier for the end user. I can go with a tabstop but that would require a more advanced template using a bunch of ternary operators and use of `null`.

Reply:
makes sense thx